third-party-podspecs/RNCMaskedView.podspec.json

```diff
@@ -2,18 +2,21 @@
   "name": "RNCMaskedView",
   "version": "0.1.10",
   "summary": "React Native MaskedView component",
+  "license": "MIT",
   "authors": "Mike Nedosekin <crespo8800@gmail.com>",
   "homepage": "https://github.com/react-native-community/react-native-masked-view#readme",
-  "license": "MIT",
   "platforms": {
-    "ios": "9.0"
+    "ios": "9.0",
+    "tvos": "9.0"
   },
   "source": {
-    "git": "https://github.com/react-native-community/react-native-masked-view.git"
+    "git": "https://github.com/react-native-community/react-native-masked-view.git",
+    "tag": "v0.1.10"
```

Comment:
There doesn't seem to be tags at all on this repo, so installing using this podspec is failing on my tests:

```
warning: Could not find remote branch v0.1.10 to clone.
fatal: Remote branch v0.1.10 not found in upstream origin
```

https://github.com/react-native-community/react-native-masked-view/tags

Reply:
> But doing a test installing this on WPiOS it failed.

hey @etoledom What kind of test did you did? I am able to install the pods after pointing the gutenberg dependency to this PR:

`gutenberg :commit => 'aedba0fc79e7d36c9652a517519350dc42975625'`

Reply:
I did this same test (tried again just now).

This is the full error 
```
Installing RNCMaskedView 0.1.10

[!] Error installing RNCMaskedView
[!] /usr/bin/git clone https://github.com/react-native-community/react-native-masked-view.git /var/folders/fc/lx8__rnx3t593pk1gyrmrkqc0000gn/T/d20200904-64255-zzeok3 --template= --single-branch --depth 1 --branch v0.1.10

Cloning into '/var/folders/fc/lx8__rnx3t593pk1gyrmrkqc0000gn/T/d20200904-64255-zzeok3'...
warning: Could not find remote branch v0.1.10 to clone.
fatal: Remote branch v0.1.10 not found in upstream origin
```

Might it be some local thing? (For you to success or for me to fail)
It does make sense to me that the tag is not found if there's no tag on the remote repo 🤔

Reply:
OK I removed the WPiOS/Pods folder and tried again, started to the same error:

```
Installing RNCMaskedView (0.1.10)

[!] Error installing RNCMaskedView
[!] /usr/bin/git clone https://github.com/react-native-community/react-native-masked-view.git /var/folders/6t/6rw82zvj7hx5brzf3kcdf7x00000gn/T/d20200904-56020-1vk5oaw --template= --single-branch --depth 1 --branch v0.1.10

Cloning into '/var/folders/6t/6rw82zvj7hx5brzf3kcdf7x00000gn/T/d20200904-56020-1vk5oaw'...
warning: Could not find remote branch v0.1.10 to clone.
fatal: Remote branch v0.1.10 not found in upstream origin

rake aborted!
Command failed with status (1): [bundle exec pod install...]

```

Reply:
I was hoping the error was on my side 😞 
Thanks for checking!

Reply:
Tag is added: https://github.com/wordpress-mobile/react-native-masked-view/tags

Problem is solved in my local but I'll open a WPiOS PR to test the CI.

Reply:
[WordPress-iOS PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/14841)